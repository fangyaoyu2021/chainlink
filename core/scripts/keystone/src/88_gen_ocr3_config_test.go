package src

import (
	"testing"

	"github.com/gkampitakis/go-snaps/snaps"
)

func TestGenerateOCR3Config(t *testing.T) {
	// Generate OCR3 config
	config := generateOCR3Config("./testdata/SampleConfig.json", 11155111, "./testdata/PublicKeys.json")
	snaps.MatchJSON(t, config)
}
