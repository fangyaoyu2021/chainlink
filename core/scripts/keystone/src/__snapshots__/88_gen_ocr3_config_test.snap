
[TestGenerateOCR3Config - 1]
{
 "F": 1,
 "OffchainConfig": "0xc80180e497d012d00180e497d012d80180a8d6b907e00180cab5ee01e80180d88ee16ff0010afa0104010101018202203ca9918cd2787de8f9aff91f220f30a5cc54c394f73e173b12c93368bd7072ad820220365b9e1c3c945fc3f51afb25772f0a5a1f1547935a4b5dc89c012f590709fefe8202207fe2dbd9f9fb96f7dbbe0410e32d435ad67dae6c91410189fe5664cf3057ef1082022091b393bb5e6bd6fd9de23845bcd0e0d9b0dd28a1d65d3cfb1fce9f91bd3d8c198202204f42ef42e5cc351dbbd79c29ef33af25c0250cac84837c1ff997bc111199d07e8a0234313244334b6f6f574e6d684b5a4c31585734567633724e6a4c587a4a366d7163566572696864696a6a475975657850724655465a8a0234313244334b6f6f5746556a5637335a596b414d6853326356777465336b584457443859627978337539434544484e6f65456842488a0234313244334b6f6f5752547448325757727a74443837446f316b586550536d476a79553472376d5a565754686d71544767646255438a0234313244334b6f6f574d545a6e5a7463564b34454a736a6b4b73563971584e6f4e52536a543632435a6933744b6b584761437347688a0234313244334b6f6f5752734d39796f7264525144684c676245724838574d4d477a316243314a34685235674147764d577538676f4e98028094ebdc03a0028094ebdc03a8028094ebdc03b0028094ebdc03ba029e010a201a92418184009e8871ee550a951ff408b0f5237c344ecaca64c1ad7d2289125d12205ec481cb0c8fb0885672b7aa1c390fd232264561a889355451d4f84b9887a2021a10c91f37b946c468994ac6fbf7309101951a10da8ed1ae8689df2bcd62bcb28bd0df471a1030ad7de593248f9d8d45dd61387f0c8b1a10f3d95715842c463e1b7db1ff098035171a10a34e73cfcde30aec56e862fc3e15f8b9c00280e497d012c8028094ebdc03",
 "OffchainConfigVersion": 30,
 "OnchainConfig": "0x",
 "Signers": [
  "0xA2402db8E549f094EA31e1C0EDd77623F4cA5b12",
  "0x4af19c802B244D1d085492C3946391C965E10519",
  "0x61925685d2B80b121537341d063c4E57B2F9323c",
  "0xFd97eFD53FC20acc098Fcd746C04d8d7540D97E0",
  "0xA0b67Dc5345a71D02B396147AE2cb75dDA63CbE9"
 ],
 "Transmitters": [
  "0xF4e7e516146c8567F8E8be0ED1f1A92798628d35",
  "0x8B60FDcc9CAC8ea476b31d17011CB204471431d9",
  "0x6620F516F29979B214e2451498a057FDd3a0A85d",
  "0xFeB61E22FCf4F9740c9D96b05199F195bd61A7c2",
  "0x882Fd04D78A7e7D386Dd5b550f19479E5494B0B2"
 ]
}
---
